api/controllers/UserController.js

```diff
@@ -3,20 +3,24 @@
  *
  * @description :: Server-side logic for managing Users
  */
-/**
- * { function_description }
- *
- * @public
- *
- * @memberof   (parent_name_path)
- *
- * @author     manoj
- *
- * @class      User (name)
- * @param      {User}  options  The options
- */
-function User(options) {
-  options = 'hello';
-}
 
-module.exports = User();
+module.exports = {
+  /**
+  	 * { function_description }
+  	 *
+  	 * @public
+  	 *
+  	 * @memberof   (parent_name_path)
+  	 *
+  	 * @author     manoj
+  	 *
+  	 * @param      {object}  req     The request
+  	 * @param      {object}  res     The resource
+  	 */
+  getdata: function getdata(req, res) {
```

Comment:
What is data in getdata?? your action (function) should be session or getSession (Note: also follow camel case)